tests/tx_validation_test.go

```diff
@@ -305,10 +305,6 @@ func TestValidationPoolInsertMagma(t *testing.T) {
 
 		txs = append(txs, tx)
 
-		if err := bcdata.GenABlockWithTransactions(accountMap, txs, prof); err != nil {
-			t.Fatal(err)
-		}
-
```

Comment:
You will change back to the original code after fixing all related bug, right? Can you just comment out and add TODO tag for this?

Reply:
resolved by e24351e